src/test/java/picard/analysis/directed/CollectHsMetricsTest.java

```diff
@@ -217,6 +217,41 @@ public void testHsMetricsHandlesIndelsAppropriately() throws IOException {
         Assert.assertEquals(insWithIndelHandling.PCT_USABLE_BASES_ON_TARGET,   1.0d);      // inserted bases are counted as on target
     }
 
+    @Test
+    public void testHsMetricsF80DoesNotUseCovCap() throws IOException {
+        final SAMRecordSetBuilder highCoverage = new SAMRecordSetBuilder(true, SortOrder.coordinate);
```

Comment:
given the code duplication between this, `testHsMetricsHandlesIndelsAppropriately` and `testHsMetricsHighTargetCoverage`, feels like there a prime refactoring opportunity here.

Reply:
Happy to do this if you feel strongly, but I'm not sure how much more maintainable the code would be for extracting a couple lines into a common method, to be honest.

Reply:
that's fair, I'm fine leaving as is.